manifests/dcgm/telegraf.pp

```diff
@@ -9,6 +9,12 @@
 # @param enable
 #   Enable or disable telegraf reporting for DCGM
 #
+# @param allownvdebugging
+#   Enable a file check to not reenable dcgm when a user is debugging
+#
+# @param nvidia_debug_check
+#   The file to check to see if user is debugging and not start the service
+#
```

Comment:
I don't see these params getting used anywhere. These left over from when you were trying a different method?

Reply:
It is remnants from a previous attempt. I will remove them.